packages/next/src/export/index.ts

```diff
@@ -410,6 +410,12 @@ async function exportAppImpl(
       authInterrupts: !!nextConfig.experimental.authInterrupts,
     },
     reactMaxHeadersLength: nextConfig.reactMaxHeadersLength,
+    hasReadableErrorStacks:
+      nextConfig.experimental.serverSourceMaps === true &&
+      (process.env.TURBOPACK
+        ? nextConfig.experimental.turbopackMinify === false
+        : nextConfig.experimental.serverMinification === false) &&
```

Comment:
The actual fix would be to implement name mapping which is tracked in https://linear.app/vercel/issue/NEXT-4417. Otherwise even unmangled code will have unreadable names for stackframes pointing to module evaluation.

So this should mention that stacks will be readable with mangled code once we fix https://linear.app/vercel/issue/NEXT-4417

I'd also argue that the gains of `--debug-prerender` are negligible once https://github.com/vercel/next.js/pull/80317 lands. The most relevant name will be in the codeframe already.